RoddyCore/test/src/de/dkfz/roddy/execution/io/BEExecutionServiceTestInlineScript.groovy

```diff
@@ -37,7 +36,7 @@ public class BEExecutionServiceTestInlineScript {
         ConfigurationFactory.initialize(LibrariesFactory.getInstance().getLoadedPlugins().collect { it -> it.getConfigurationDirectory() })
 
         final Configuration mockupConfig = new Configuration(new PreloadedConfiguration(null, Configuration.ConfigurationType.OTHER, "test", "", "", null, "", ResourceSetSize.l, null, null, null, null), ConfigurationFactory.getInstance().getConfiguration("default"))
-        mockedContext = MockupExecutionContextBuilder.createSimpleContext(BEExecutionServiceTest, mockupConfig);
+        mockedContext = ContextResource.createSimpleContext(BEExecutionServiceTest, mockupConfig);
```

Comment:
If the class already extends ContextResource, why do you need to access ContextResource ?